src/backend/HostedFileStorageSystemClient.js

```diff
@@ -21,7 +21,7 @@ export default {
     const token = Cookies.get(ACCESS_TOKEN_COOKIE_KEY);
     try {
       const response = await fetchWithTimeout(
-        SERVER_BASE_URL + `view?docID=${sourceId}` + (!createNew ? `&viewID=${viewId}&type=${type}` : ''),
+        SERVER_BASE_URL + `view?docID=${sourceId}&viewType=${type}` + (!createNew ? `&viewID=${viewId}` : ''),
```

Comment:
Ah, I guess these changes relate to your other PR

Reply:
Yes, the url doesnt take viewID for createNew